HW_14/task_3.py

```diff
@@ -0,0 +1,26 @@
+def arg_rules(type_: type, max_length: int, contains: list):
+    def decoraror(func):
+        def wrapper(*args, **kwargs):
+            my_string = func(*args, **kwargs)
+            print(my_string)
+            for item in args:
+                if isinstance(item, type_) and len(item) <= max_length and not contains in args:
```

Comment:
Wrong logic for `contains` here. Basically you are checking that the list is present as one of the elements in `args`